manifests/application/migrator.pp

```diff
@@ -22,31 +26,31 @@
 # $primary_client_metadata_uri       - see lightblue::client::configure
 # $primary_client_data_uri           - see lightblue::client::configure
 # $primary_client_use_cert_auth      - see lightblue::client::configure
-# $primary_client_ca_file_path       - see lightblue::client::configure
+# $primary_client_ca_file_path       - (optional) ca file path
 # $primary_client_ca_source          - (optional) source for ca file.
-# $primary_client_cert_file_path     - see lightblue::client::configure
+# $primary_client_cert_file_path     - (optional) cert file path
 # $primary_client_cert_source        - (optional) source for cert file.
 # $primary_client_cert_password      - see lightblue::client::configure
 # $primary_client_cert_alias         - see lightblue::client::configure
 #
-# $source_config                     - Source lightblue client configuration file. Defaults to $config_file.
+# $is_source_same_as_primary         - Indicates if the source lightblue instance is the same as the primary. Defaults to true.
```

Comment:
Previously the assumption was made that the primary, source, and destination config files would always be provided if they were being used. Now it is more likely that puppet will be generating and landing those configuration files.